projects/ngx-charts-on-fhir/src/lib/fhir-chart/range-selector/range-selector.component.css

```diff
@@ -11,3 +11,15 @@
 .range-selector mat-form-field {
   width: 160px;
 }
+
+.display-flex {
+  display: flex;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.gap-8 {
+  gap: 8px;
+}
```

Comment:
This is an anti-pattern. Generally speaking, CSS classes should be semantic. They should represent a design element (or one aspect/state of an element). "summary-card-table" is a good example of a semantic CSS class.

"Utility" classes that set a single property should be kept to a minimum because they become difficult to maintain. We would need to change both the CSS and the HTML every time we want to change one of these properties.